guidelines/data.md

```diff
@@ -0,0 +1,20 @@
+**_Journal of Cheminformatics_ will only publish research or software that is entirely reproducible by third parties.
+This means that any datasets, software and algorithms that are required to reach the conclusions stated in the
+paper must be provided as supplemental materials, or be otherwise accessible without the need for registration,
```

Comment:
Does the clause re registration or agreement with only OSS license for software applied to every submission? I think we are violating this clause when commercial software is being used. Should this be relaxed?

Reply:
When people use commercial software, we always expect it to be able to done with open tools too. So, I don't think we need to relax.